<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGroupWindow</class>
 <widget class="QWidget" name="CreateGroupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame{
background-color: rgb(255, 255, 255);
border-radius:10px;
border:1px solid rgb(100, 169,139);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>189</x>
        <y>60</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>创建群聊</string>
      </property>
     </widget>
     <widget class="QWidget" name="head" native="true">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>482</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#head{
background:rgb(0,188,212);
}</string>
      </property>
      <widget class="QToolButton" name="min_btn">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>0</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton:hover
{
	background-color:rgba(200, 200, 200,85);
}
QToolButton:pressed
{
	background-color:rgba(220,220,220,85);
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="close_btn">
       <property name="geometry">
        <rect>
         <x>441</x>
         <y>0</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton:hover 
{
	background-color:rgb(255, 0, 0);
}
QToolButton:pressed
{
	background-color:rgb(170, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="group_name">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>群名：</string>
      </property>
     </widget>
     <widget class="ElaLineEdit" name="group_name_edit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QtMaterialRaisedButton" name="send_btn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>570</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>创建</string>
      </property>
     </widget>
     <widget class="QListView" name="friend_view">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>431</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ElaLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">elalineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialraisedbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
